RELEASE_NOTES.md

```diff
@@ -0,0 +1,5 @@
+#### 1.0.1 May 27 2016 ####
+Removed non-generic IPublisher and ISubscriber interfaces.
```

Comment:
This is not binary compatible release and cannot be done in a minor version.
I am surprised that you are opening a PR when the release has already been made.
The purpose of opening PRs is to review changes *before* they are released, not post-facto.